core/policy.go

```diff
@@ -48,8 +48,8 @@ type DefaultQueuePolicy struct {
 
 // NewMessageQueuePolicy returns a new policy which removes mined messages from the queue and expires
 // messages older than `maxAgeTipsets` rounds.
-func NewMessageQueuePolicy(messages MessageProvider, maxAge uint64) *DefaultQueuePolicy {
-	return &DefaultQueuePolicy{messages, maxAge}
+func NewMessageQueuePolicy(messages MessageProvider, maxAge uint) *DefaultQueuePolicy {
```

Comment:
why was this converted to a uint when it gets cast immediately to uint64?

Reply:
The appropriate interface type is uint, since it's never going to be large. This is to match the inbox parameter that does the same thing. As an implementation detail, this one converts to uint64 for arithmetic later on.